Open Judge System/OJS.Common/Models/ExecutionStrategyType.cs

```diff
@@ -34,5 +34,6 @@ public enum ExecutionStrategyType
         MySqlPrepareDatabaseAndRunQueries = 16,
         MySqlRunQueriesAndCheckDatabase = 17,
         MySqlRunSkeletonRunQueriesAndCheckDatabase = 18,
+        RubyExecutionStrategy = 32
```

Comment:
what is the arrangement of these Strategies? It is neither by value, nor alphabetically by name...